app/models/concerns/better_together/hosts_events.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module BetterTogether
+  # Concern that when included gives the model access to events through event_host records
+  # This module must be included in a model to permit assigning instances as an event host
+  module HostsEvents
```

Comment:
@Kneeba921 include this module in https://github.com/NL-Venues/nl-venues/blob/feature/event-hosts/app/models/venue.rb to allow-list the class as a permitted event host type